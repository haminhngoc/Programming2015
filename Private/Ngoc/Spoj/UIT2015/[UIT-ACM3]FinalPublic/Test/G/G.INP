42 68 35 501
70 25 79 359
63 65 6 146
82 28 62 492
96 43 28 437
92 5 3 154
93 83 22 717
19 96 48 727
72 39 70 913
68 100 36 895