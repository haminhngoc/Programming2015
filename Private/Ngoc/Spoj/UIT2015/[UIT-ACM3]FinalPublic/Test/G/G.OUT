897862140
1786743000
87886890
4316041464
2765433888
16470300
43711662654
482681472
13668815160
6134688000